static/app/views/settings/organizationDeveloperSettings/sentryFunctionDetails.tsx

```diff
@@ -57,30 +76,53 @@ function SentryFunctionDetails(props: Props) {
 
   const handleSubmitSuccess = data => {
     addSuccessMessage(t('Sentry Function successfully saved.', data.name));
+    const baseUrl = `/settings/${orgId}/developer-settings/sentry-functions/`;
+    // TODO: should figure out where to redirect this
+    const url = `${baseUrl}${data.slug}/`;
+    if (sentryFunction) {
+      addSuccessMessage(t('%s successfully saved.', data.name));
+    } else {
+      addSuccessMessage(t('%s successfully created.', data.name));
+    }
+    browserHistory.push(url);
   };
 
   function handleEditorChange(value, _event) {
     form.current.setValue('code', value);
   }
 
-  const defaultCode = `exports.yourFunction = (req, res) => {
-    let message = req.query.message || req.body.message || 'Hello World!';
-    console.log('Query: ' + req.query);
-    console.log('Body: ' + req.body);
-    res.status(200).send(message);
-};`;
+  async function handleDelete() {
+    // figure out how to send a delete request
+    const response = await api.requestPromise(endpoint, {
+      method: 'DELETE',
+    });
+    if (response.status === 204) {
+      browserHistory.push(`/settings/${orgId}/developer-settings/`);
+      // TODO: Maybe show a success message?
+    } else if (response.status >= 400 && response.status < 500) {
```

Comment:
@vpandiarajan20 does this actually work? I would expect `requestPromise` to throw an error if there is an error status code preventing it from hitting this line